### PR TITLE
[NEW SEARCH] World level roads to new-search branch

### PR DESCRIPTION
**Pull request by gardster**

### CODE REVIEW

**Comment by mpimenov** [comment]
LGTM

File: generator/towns_dumper.hpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      if (key == "population")
+      {
+        try
+        {
+          strings::to_uint64(value, population);
+        }
+        catch (std::invalid_argument const &)
+        {
+          continue;
+        }
```
Comment (vng) [inline generator/towns_dumper.hpp]: if (!strings::to_uint64)
  continue;

**Comment by gardster** [comment]
PTAL

**Comment by gardster** [comment]
DUDE
JTTP

**Comment by gardster** [comment]
DUDE
JTTP

**Comment by gardster** [comment]
DUDE
JTTP

**Comment by vng** [comment]
LGTM

**Pull request merged**